velox/docs/functions/presto/math.rst

```diff
@@ -27,6 +27,15 @@ Mathematical Functions
     verified for performance reasons. Returns ``high`` for all values of ``x``
     when ``low`` is greater than ``high``.
 
+.. function:: cosine_similarity(map(varchar, double), map(varchar, double)) -> double
+
+    Returns the cosine similarity between the sparse vectors x and y. If any value of any key is NULL,
+    the function returns NULL.
+
+    For example:
```

Comment:
The formatting here might be off. Please, generate docs locally and verify. Would be great to share a screenshot.

Reply:
That would be nice. CC: @kgpai